Swat/SwatTextareaEditor.php

```diff
@@ -349,6 +354,51 @@ protected function getInlineJavaScript()
         // Make removeformat button also clear inline alignments, styles,
```

Comment:
this comment looks out of date. Can you update it and also include a description of what you're doing in the post-paste step?